charts/rollups-validator-node/values.yaml

```diff
@@ -124,6 +124,87 @@ validator:
   ##
   extraVolumes: []
 
+  localnode:
+    enabled: false
+    storage:
+      blockchainData:
+        storageClass: standard
+        size: 2Gi
+      deployments:
+        storageClass: standard
+        size: 1Gi
+      machineSnapshots:
+        storageClass: standard
+        size: 2Gi
+    service:
+      type: ClusterIP
+      port: 8545
+
+    anvil:
+      enabled: false
+      image:
+        registry:
+        repository: sunodo/anvil
+        tag:
+        digest:
+
+      command: []
+
+      args: []
+      extraArgs: []
+
+      ## extraEnvVars Array with extra environment variables to add to validator.anvil container
+      ## e.g:
+      ## extraEnvVars:
+      ##   - name: FOO
+      ##     value: "bar"
+      ##
+      extraEnvVars: []
+
+      ## extraEnvVarsCM Name of existing ConfigMap containing extra env vars for anvil container
+      ##
+      extraEnvVarsCM: ""
+
+      ## extraEnvVarsSecret Name of existing Secret containing extra env vars for anvil container
+      ##
+      extraEnvVarsSecret: ""
+
+      securityContext:
+        {}
+        ## capabilities:
+        ##   drop:
+        ##   - ALL
+        ## readOnlyRootFilesystem: true
+        ## runAsNonRoot: true
+        ## runAsUser: 1000
+
+      resources:
+        {}
+        ## limits:
+        ##   cpu: 100m
+        ##   memory: 128Mi
+        ## requests:
+        ##   cpu: 100m
+        ##   memory: 128Mi
+
+      ## validator.anvil.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the anvil container(s)
+      ##
+      extraVolumeMounts: []
+
+      anvilState:
+        image:
+          registry:
+          repository: sunodo/rollups-node
+          tag:
+          digest:
```

Comment:
If it's just for the initContainer that makes a simple `cp`, let's use another image, like `alpine` and remove this.

```suggestion
```

Reply:
I use the `sunodo/rollups-node` filesystem to copy below to the `blockchain-data` PVC:

- anvil_state.json
 - localhost.json

Reply:
so if there's the requirement to be the sunodo/rollups-node image, we could hardcode that, right?

Do you see any case that someone would need to use a different image?

Reply:
I thought Maybe  in the future, we will have another tag or another image name

Reply:
If that's the case, we update the chart with a minor/patch release.

The same happens when we need to bump the version for rollups, even if there's no breaking changes, we need to bump the chart version.

Reply:
Got it !

Reply:
Fixed !